starter-code/main.js

```diff
@@ -0,0 +1,284 @@
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "7":
+						db.collection('companies').find({}, {name: 1, number_of_employees: 1, _id: 0})
+						.sort({number_of_employees: -1})
+						.limit(10).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "8":
+						db.collection('companies').find({name: "Facebook"}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "9":
+						db.collection('companies').find({name: "Facebook"}, {number_of_employees: 1, _id: 0}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "10":
+						db.collection('companies').find({name: "Facebook"}, {"products.name": 1, _id: 0}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "11":
+						db.collection('companies').find(
+								{name: "Facebook", "relationships.is_past": false},
+								{"relationships": 1, _id: 0}
+							).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								// console.log(JSON.stringify(result));
+								// This is a correction
+								result[0].relationships.filter((person) => {
+									return person.is_past === false;
+								}).forEach((worker) => {
+									console.log(`${worker.person.first_name} ${worker.person.last_name}`);
+								});
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "12":
+						db.collection('companies').find({name: "Facebook", "relationships.is_past": true}).count((error, result) => {
```

Comment:
This won't work. In a similar way to before, use a filter after the query and return the length of the filtered array.